src/js/components/Table/Table.js

```diff
@@ -157,7 +165,14 @@ class Table extends PureComponent {
       <Panel className={ className } mods={ mods } isStriped={ isStriped } style={ style } { ...otherProps }>
         <PanelBody>
           <PanelRow isWithCells>{ this.renderTableColumns() }</PanelRow>
-          { this.renderTableRows() }
+          {otherProps.scrollTableBody &&
```

Comment:
This is probably not the way to go about this.   The otherProps are used to spread out on the parent component as you can see above `{ ...otherProps }`.  In this case, you'll probably want to add a new prop for this.

Reply:
maybe if a set table height is passed, enable the scrollable table?   As long as if the table is less than the height it wouldn't be messed up?